src/System.Net.Primitives/src/MatchingRefApiCompatBaseline.netcoreapp.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Net.Primitives:
+TypesMustExist : Type 'System.Net.CookieVariant' does not exist in the implementation but it does exist in the contract.
```

Comment:
@davidsh can you please confirm whether these should be exposed in the reference assembly?

Reply:
Please just baseline this for now.